### DISCUSSION TITLE
SAKAI-NG CSS Layer Management

### DISCUSSION THREAD

**Discussion by The-Real-1** [primefaces/community#588] [PrimeNG Templates] [answered]
Hi,

We are using sakai-ng to develop our PrimeNg application theme, as it's now the new way to do.

Sakai comes with a bunch of scss files (visible in src\assets\layout\styles\layout*), and this is ok.

The problem is those file are considered as anonymous layers and they have a bigger priority in the rendering process.
For example, using css custom layers while templating the h1 tag will never be rendered as expected because the applied rule will be the one specified in _typography.scss.

Adding a layer sakai definition for all files in src\assets\layout\styles\layout will be, I think, a good idea to prevent this behavior and let teams render has expected their styles.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We'll upgrade the next-gen templates to v17. Sakai will receive a v17 update, and we'll address this issue.

**Reply by The-Real-1** [reply]
Hi,

That's a great news. Thanks for the reply.
